tests/components/elevenlabs/test_tts.py

```diff
@@ -490,3 +491,144 @@ async def test_tts_service_speak_without_options(
         ),
         model_id="model1",
     )
+
+
+async def test_tts_supports_streaming_input(
+    hass: HomeAssistant,
+    config_data: dict[str, Any],
+    config_options: dict[str, Any],
+) -> None:
+    """Test that the TTS entity supports streaming input."""
+    await mock_config_entry_setup(hass, config_data, config_options)
+    await hass.async_block_till_done()
+
+    tts_entity = hass.data[tts.DOMAIN].get_entity("tts.mock_title")
+    assert tts_entity.async_supports_streaming_input() is True
+
+
+async def test_tts_streaming_audio(
+    hass: HomeAssistant,
+    config_data: dict[str, Any],
+    config_options: dict[str, Any],
+) -> None:
+    """Test streaming TTS audio generation."""
+    await mock_config_entry_setup(hass, config_data, config_options)
+    await hass.async_block_till_done()
+
+    tts_entity = hass.data[tts.DOMAIN].get_entity("tts.mock_title")
+
+    # Mock the streaming audio response
+    mock_audio_stream = FakeAudioGenerator()
+    tts_entity._client.text_to_speech.convert = MagicMock(
+        return_value=mock_audio_stream
+    )
+
+    # Create a mock message generator
+    async def message_generator():
+        yield "Hello, "
+        yield "this is a "
+        yield "test message."
+
+    # Create a mock TTSAudioRequest
+    request = TTSAudioRequest(
+        language="en", options={}, message_gen=message_generator()
+    )
+
+    # Test streaming TTS
+    response = await tts_entity.async_stream_tts_audio(request)
```

Comment:
we should use the `tts.async_create_stream` method instead of interacting directly with the entity, like wyoming tests do